packages/grafana-ui/src/components/Logs/LogRows.tsx

```diff
@@ -98,7 +102,10 @@ class UnThemedLogRows extends PureComponent<Props, State> {
       ? dedupedRows.reduce((sum, row) => (row.duplicates ? sum + row.duplicates : sum), 0)
       : 0;
     const showDuplicates = dedupStrategy !== LogsDedupStrategy.none && dedupCount > 0;
-    const horizontalScrollWindow = wrapLogMessage ? '' : logsRowsHorizontalScroll;
+
+    // For horizontal scrolling we can't use CustomScrollbar as it causes the problem with logs context - it is not visible
+    // for top log rows. Therefore we use CustomScrollbar only in LogsPanel and for Explore, we use custom css styling.
+    const horizontalScrollWindow = wrapLogMessage && !logsPanel ? '' : logsRowsHorizontalScroll;
```

Comment:
So what will happen if we are in logsPanel and somebody selects `wrapLogMessges=true`? Does this basically disable that option in Logs Panel?

Reply:
No, it still works, because we are using CustomScrollbar in LogsPanel that can handle horizontal and vertical scrolling.
![Kapture 2020-05-19 at 17 27 25](https://user-images.githubusercontent.com/30407135/82345865-0d28be80-99f6-11ea-8346-fc848befaccf.gif)

Reply:
But this brought be to another question - should we do autoHide for CustomScrollbar in Logs Panel? Maybe without autoHide it is going to be more clear that you can scroll both. directions?

Reply:
Cool in that case though I would rename the prop. `logsPanel` does not say what the prop means, it says that parent is logsPanel which is something the component should not care/know. Something like `disableHorizontalScroll` or something like that would make more sense.

> should we do autoHide for CustomScrollbar in Logs Panel

Looking at the usages it seems to be mixed. Personally hate autoHide for scrolling and is the first thing I disable in mac :D. So I would vote to remove it. is that white background of the scroll bar something you did extra? Cause there is no background on my machine.

Reply:
> cool in that case though I would rename the prop. 

Yeah, good point. 

> I would vote to remove it.  

I don't have strong preference. Maybe we can keep it the way it is and disable/leave it enabled when/if we have strategy for that. 

>is that white background of the scroll bar something you did extra?

No that is my Chrome's default scrollbar's background. It looks normal and nice but changes to that ugly one when I'm over it. It is the same on every website. 
<img width="1440" alt="Screenshot 2020-05-19 at 17 47 21" src="https://user-images.githubusercontent.com/30407135/82348176-cd170b00-99f8-11ea-869a-be8f078efa90.png">
<img width="1920" alt="Screenshot 2020-05-19 at 17 47 21 (2)" src="https://user-images.githubusercontent.com/30407135/82348198-d1432880-99f8-11ea-90a3-d7a910c98d23.png">

![image](https://user-images.githubusercontent.com/30407135/82348025-9e993000-99f8-11ea-92be-a73987fda167.png)